src/controllers/favorites.controllers.js

```diff
@@ -0,0 +1,17 @@
+import { StatusCodes } from 'http-status-codes';
+import { addToFavorite } from '../services/favoritesService';
+
+const favoritesControllers = (router) => {
+  router.post('/favorites', async (req, res) => {
+    const response = await addToFavorite(req.body); // {userId: '123', houseId: '321'}
+    if (!response.status) {
```

Comment:
I suggest adding `?` in case the response is undefined. It will prevent unhandled rejection. (null instead of undefined) 
```suggestion
    if (!response?.status) {
```

Reply:
![obraz](https://user-images.githubusercontent.com/80857044/154820328-588d9b5d-7546-414e-b524-58f395dec19b.png)

Reply:
Sorry that I didn't check that out. I think there are some configuration problems - I already removed the chaining operator from my code. 

```suggestion
    if (!response || !response.status)
```